fbsimctl/FBSimulatorControlKit/Sources/Command.swift

```diff
@@ -83,6 +83,7 @@ public enum Action {
   case Uninstall(String)
   case Upload([FBDiagnostic])
   case WatchdogOverride([String], NSTimeInterval)
+  case setLocation(Double,Double)
```

Comment:
While newer versions of swift don't have capitalization of their first character, this is inconsistent with the rest of the Actions. When we do migrate to using the new syntax, we can do it all at once. Please make the first character uppercase to be consistent with the rest of the actions.